stable/loki/templates/loki/configmap.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "loki.fullname" . }}
+  labels:
+    app: {{ template "loki.name" . }}
+    chart: {{ template "loki.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  loki.yaml: |
+    auth_enabled: {{ .Values.loki.config.auth_enabled }}
+
+    server:
+      http_listen_port: {{ .Values.loki.service.port }}
+
+    limits_config:
+      enforce_metric_name: false
+
+    ingester:
+      lifecycler:
+        ring:
+          store: {{ .Values.loki.config.ingester.lifecycler.ring.store }}
+          replication_factor: {{ .Values.loki.config.ingester.lifecycler.ring.replication_factor }}
+      chunk_idle_period: 15m
+
+{{- if .Values.loki.config.schema_configs }}
+    schema_config:
+      configs:
+  {{- range .Values.loki.config.schema_configs }}
```

Comment:
Is there a reason not to just 'toYaml .' inside the range rather then build out each item explicitly?

Reply:
I saw some code use `toYaml` while some not, i did not know in which case use it or not.